src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -198,17 +197,39 @@ internal static void UpdateBackground(this EditText platformView, IView view)
 				mauiDrawable.Dispose();
 			}
 
-			// Android will reset the padding when setting a Background drawable
+			if (platformView.Background is LayerDrawable layerDrawable)
+			{
+				platformView.Background = null;
+				layerDrawable.Dispose();
+			}
+
+			// Android will reset the padding when setting a Background drawable	
 			// So we need to reapply the padding after
 			var padLeft = platformView.PaddingLeft;
 			var padTop = platformView.PaddingTop;
 			var padRight = platformView.PaddingRight;
 			var padBottom = platformView.PaddingBottom;
 
-			var previousDrawable = platformView.Background;
+			var paint = view.Background;
+
+			Drawable? defaultBackgroundDrawable = ContextCompat.GetDrawable(platformView.Context, Resource.Drawable.abc_edit_text_material);
+
+			var previousDrawable = defaultBackgroundDrawable ?? platformView.Background;
 			var backgroundDrawable = paint!.ToDrawable(platformView.Context);
```

Comment:
I'm still not quite following the null forgiveness operators on this PR. 

`view.Background` returns `Paint?`

![image](https://user-images.githubusercontent.com/5375137/229890775-f1a6354a-54b9-4f44-b450-a78514ce1fae.png)

It seems unsafe to just add `paint!` here instead dealing with a null paint. 

Looking at `PaintExtensions.ToDrawable` it looks like that can return null, so maybe the input parameter on that extension method should get changed to 

```C#
public static Drawable? ToDrawable(this Paint? paint, Context? context)
```
Then we don't have to add null forgiveness?

I feel like this block of code can be reworked so we null have any null forgiveness operators.

Reply:
I updated the extension method to get the drawable, could you review it again?. Thanks!